tools/ang_quad_plotting/plot_quadrature.py

```diff
@@ -0,0 +1,93 @@
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.art3d as art3d
+from pathlib import Path
+import numpy as np
+import math
+
+# Declare File Base to read
+fileBase = "TestQuad1_"
```

Comment:
when is there a 1 in that basename? the ready states that by default the basename does not contain a 1.